### DISCUSSION TITLE
End-to-End Encryption for Sync (Self+Web Hosting)

### DISCUSSION THREAD

**Discussion by VictorVow** [athensresearch/athens#868] [Archive]
I'd prefer not to self-host as it's a hassle so I'd like to use web hosting when it's available, but I'd only be comfortable using it if my data was client-side encrypted. 

An E2EE app I use frequently uses [Etebase](https://github.com/etesync/server) for this so this might be suitable for Athens too. For multimedia there's also [FileSafe](https://github.com/standardnotes/filesafe-relay) which allows files to be uploaded to popular cloud storage (Google Drive, Dropbox) or WebDAV servers also.

**Comment by jsmorabito** [comment]
now on roadmap, #873 , discussion can continue on that issue. Thanks for the request Victor239!
